### ISSUE TITLE
Can't run Command-line tool on iTerm2 with Apple Silicon Mac

### ISSUE THREAD

**Issue by 0x0c** [nicklockwood/SwiftFormat#823]
Hi, I got problem to run command-line tool installed the tool from CocoaPods when using `iTerm2` and Apple Silicon Mac.
When I ran `./swiftformat`, the following message is shown.
```
[1]    31670 killed     ./swiftformat
```
However, there is no problem when using `Terminal`.
Both terminals run with arm64 (doesn't using Rosetta2).
Here is my environment.
```
- iTerm2 3.4.2
- macOS 11.0.1
- zsh 5.8 (x86_64-apple-darwin20.0) (with oh-my-zsh) 
- SwiftFormat 0.47.5
```

Does anyone knows any workaround?

**Comment by nicklockwood** [comment]
@0x0c this is very weird. I thought it might be something to do with ANSI characters, but none are used in the help screen that is displayed when you run without arguments AFAIK.

I guess it's worth trying `./swiftformat --quiet` just in case it is something like that?

**Comment by 0x0c** [comment]
@nicklockwood Thank you for your reply.
I tried `./swiftformat --quet` and still can't run.
Here are results each environment, `iTerm2`, `iTerm2 (with Rosetta2)` and `Terminal`.
Do you have any suggestion for me to investigate this issue? 
<img width="710" alt="Screen Shot 2020-12-07 at 18 53 26" src="https://user-images.githubusercontent.com/154327/101336604-042e0a80-38be-11eb-8aca-e4319435a49a.png">
<img width="719" alt="Screen Shot 2020-12-07 at 18 53 32" src="https://user-images.githubusercontent.com/154327/101336621-08f2be80-38be-11eb-82c2-172b9e5adcb8.png">
<img width="698" alt="Screen Shot 2020-12-07 at 18 53 21" src="https://user-images.githubusercontent.com/154327/101336697-1b6cf800-38be-11eb-9599-9c877e971054.png">

**Comment by nicklockwood** [comment]
@0x0c sorry, I'm not really sure how to debug this. Is there some kind of crash log? Can you build a debug version from source and run it with the debugger attached?

**Comment by 0x0c** [comment]
@nicklockwood I found an error message when `./swiftformat --quiet` crashes from Console.app (There is typo but can't run).

<img width="657" alt="Screen Shot 2020-12-07 at 19 10 02" src="https://user-images.githubusercontent.com/154327/101338094-dba71000-38bf-11eb-9484-3b06805366b8.png">

After got this message, I guessed this issue is caused by settings of permission, then I allowed full disk access to iTerm2 and `./swiftformat` ran!

I think there are two ways to solve this issue, one is sign the binary, and other is enable full disk access for iTerm2.
It may helpful to notice this workaround in README.

**Comment by nicklockwood** [comment]
@0x0c ah, that makes sense. I normally install using homebrew, which I guess handles signing correctly. I'll make sure the binary is signed for the next release.

**Comment by 0x0c** [comment]
@nicklockwood I got it. Thanks a lot!

**Comment by nicklockwood** [comment]
@0x0c fixed in 0.47.6
